Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -551,22 +551,22 @@ section NumberField
 variable [NumberField K]
 
 /-- The number of infinite real places of the number field `K`. -/
-noncomputable abbrev NrRealPlaces := card { w : InfinitePlace K // IsReal w }
+noncomputable abbrev nrRealPlaces := card { w : InfinitePlace K // IsReal w }
 
 /-- The number of infinite complex places of the number field `K`. -/
```

Comment:
```suggestion

@[deprecated (since := "2024-10-24")] alias NrRealPlaces := nrRealPlaces

/-- The number of infinite complex places of the number field `K`. -/
```